### PR TITLE
[one-cmds] Disable one-import-onnx-ext with env.var.

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise to disable one-import-onnx-ext with environment variable.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for https://github.com/Samsung/ONE/issues/12872#issuecomment-2177452791

File: compiler/one-cmds/one-import-onnx [hunk 1]
```diff
@@ -247,6 +247,9 @@ def _remap_io_names(onnx_model):
 def _check_ext(args):
     if oneutils.is_valid_attr(args, 'disable_ext'):
         return None
+    env_disable_ext = os.getenv('ONE_IMPORT_ONNX_EXT_DISABLE')
+    if env_disable_ext == '1' or env_disable_ext == 'Y':
```
Comment (shs-park) [inline compiler/one-cmds/one-import-onnx:251]: What if we make it work case-insensitively =)

```suggestion
    if env_disable_ext == '1' or env_disable_ext.upper() == 'Y':
```

**Review by shs-park — DISMISSED** [review]
I left a suggestion, PTAL

**Comment by seanshpark** [comment]
@shs-park , applied :) PTAL again.

**Review by shs-park — APPROVED** [review]
LGTM!
Thanks =)

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**